visualization/app/codeCharta/util/usageDataTracker.ts

```diff
@@ -248,7 +248,7 @@ export function trackEventUsageData(actionType: string, state: State, payload?:
 	}
 }
 
-function buildEventTrackingItem(actionType: string, payload?: any): EventTrackingItem | null {
```

Comment:
This is much more concrete as the previous version, will this not limit current/further functionality?

Reply:
We should never use `any`. Otherwise there's no point in using TypeScript :)

If we change a type, we have to make sure it works as expected everywhere and that's only possible by passing through the expected type.